### ISSUE TITLE
SyntaxError in function calls with two array spreads.

### ISSUE THREAD

**Issue by manuel-colmenero** [prettier/plugin-php#1976]
**Environment:**

Prettier version: 2.6.2
Plugin PHP Version: 0.18.4

**How to reproduce:**

```php
$a = [1, 2, 3];
$b = [4, 5, 6];
var_dump(...$a, ...$b);
```

**Expected ouput:**

no errors.

**Actual output:**
```
SyntaxError: Unexpected argument after a variadic argument on line 3
[error]   1 |          $a = [1, 2, 3];
[error]   2 |          $b = [4, 5, 7];
[error] > 3 |          var_dump(...$a, ...$b);
[error]     |                              ^
```

**Comment by czosel** [comment]
Fixed in `v0.18.8` :tada:

**Comment by manuel-colmenero** [comment]
This is great, thanks!
